src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs

```diff
@@ -55,57 +61,196 @@ internal static partial class JsonWriterHelper
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private static bool NeedsEscaping(char value) => value > LastAsciiCharacter || AllowList[value] == 0;
 
-        public static int NeedsEscaping(ReadOnlySpan<byte> value, JavaScriptEncoder encoder)
+#if BUILDING_INBOX_LIBRARY
+        private static readonly Vector128<short> s_mask_UInt16_0x20 = Vector128.Create((short)0x20); // Space ' '
+
```

Comment:
Nit: remove that empty line, same below for "Tilde". Then the vectors for `char` (respectively `short`) and `sbyte` are in blocks.
Or are these empty lines intentional?

Reply:
I put them there intentionally as they mark the "less than" and "greater than" characters that act as the bounds (they themselves don't need to be encoded). The others are characters that need to be encoded if found. However, I don't feel too strongly about it and can remove the extra lines.